### PR TITLE
Improve FR #'s 22, 13, 3, Task #64

### PR DESCRIPTION
**Pull request by racecn**
Finally fixed user search to search ALL users
Added search mini card to user search
added more comments
fixed unfollowing not working
began work on cropping images before being uploaded to our server
Profile picture fetching simplification
move searchbars back to top of pages

### CODE REVIEW

File: src/components/MiniUserCard.js
Comment (rishithab48) [inline src/components/MiniUserCard.js:1]: I checked this mini user card. Its working well and searched up all the users I created.

File: src/components/Searchbar.js
Comment (rishithab48) [inline src/components/Searchbar.js:1]: Search is working but the search button is giving me errors.

Comment (racecn) [reply src/components/Searchbar.js:1]: Should work fine now

**Review by rishithab48 — APPROVED** [review]
Everything looks great,

**Pull request merged**